tinyagentos/routes/a2a_bus.py

```
@@ -118,3 +125,82 @@ async def bus_messages(request: Request, channel: str = "", limit: int = 100):
+    if body.reply_to is not None:
+        payload["reply_to"] = body.reply_to
+
+    bus = _bus_url()
+    try:
+        async with httpx.AsyncClient(timeout=5.0) as client:
+            resp = await client.post(f"{bus}/a2a/send", json=payload)
+            resp.raise_for_status()
+            data = resp.json()
+    except Exception as exc:  # noqa: BLE001
+        logger.warning("A2A bus send failed (%s): %s", bus, exc)
+        raise HTTPException(status_code=502, detail="a2a bus unavailable")
```

Ruff (0.15.20)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)